### PR TITLE
Update stop sign sample to latest version

### PR DESCRIPTION
**Pull request by jwood803**
Update sample to have the latest ML.NET packages and use the latest version of .NET (.NET 5).

### CODE REVIEW

**Comment by jwood803** [comment]
@luisquintanilla @briacht Should we add a README for this sample?

**Comment by briacht** [comment]
@jwood803 yes all samples should have a README I think

**Comment by jwood803** [comment]
@briacht I'll work on it and add it for review. Thanks!

File: samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetection/StopSignDetection.csproj [hunk 1]
```diff
@@ -2,22 +2,19 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.5.0" />
+    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.6.0" />
```
Comment (luisquintanilla) [inline samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetection/StopSignDetection.csproj:9]:
```suggestion
    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="$(MicrosoftMLVersion)" />
```

File: samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetectionML.Model/StopSignDetectionML.Model.csproj [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net5</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.ML" Version="1.5.0" />
-    <PackageReference Include="Microsoft.ML.OnnxTransformer" Version="1.5.0" />
-    <PackageReference Include="Microsoft.ML.OnnxRuntime" Version="1.3.0" />
+    <PackageReference Include="Microsoft.ML" Version="1.6.0" />
+    <PackageReference Include="Microsoft.ML.OnnxTransformer" Version="1.6.0" />
```
Comment (luisquintanilla) [inline samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetectionML.Model/StopSignDetectionML.Model.csproj:8]:
```suggestion
    <PackageReference Include="Microsoft.ML" Version="$(MicrosoftMLVersion)" />
    <PackageReference Include="Microsoft.ML.OnnxTransformer" Version="$(MicrosoftMLVersion)" />
```

**Review by luisquintanilla — APPROVED** [review]
Use Directory.build.props for version #

**Comment by jwood803** [comment]
@luisquintanilla @briacht I believe this is now ready for review

**Review by luisquintanilla — APPROVED** [review]
LGTM

File: samples/modelbuilder/ObjectDetection_StopSigns/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+You can download the dataset [here](https://aka.ms/mlnet-object-detection-tutorial-assets). The download includes of the raw image data files.
+
+## The model
+
+The goal of the application is to predict whether an image includes a stop sign and, if any are found, gives their bounding boxes to where the stop sign is located in the image. The Machine Learning Task to use in this scenario is multiclass classification. The model in this application was trained using Model Builder.
+
+[Model Builder](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-model-builder) is an intuitive graphical Visual Studio extension to build, train, and deploy custom machine learning models.
+
+Model Builder Azure Machine Learning to train the object detection model.
```
Comment (luisquintanilla) [inline samples/modelbuilder/ObjectDetection_StopSigns/README.md:32]:
```suggestion
Model Builder uses Azure Machine Learning to train the object detection model.
```

File: samples/modelbuilder/ObjectDetection_StopSigns/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+The Stop Sign dataset consists of 50 images downloaded from Unsplash, each of which contain at least one stop sign.
+
+You can download the dataset [here](https://aka.ms/mlnet-object-detection-tutorial-assets). The download includes of the raw image data files.
+
+## The model
+
+The goal of the application is to predict whether an image includes a stop sign and, if any are found, gives their bounding boxes to where the stop sign is located in the image. The Machine Learning Task to use in this scenario is multiclass classification. The model in this application was trained using Model Builder.
+
+[Model Builder](https://docs.microsoft.com/en-us/dotnet/machine-learning/how-to-guides/install-model-builder) is an intuitive graphical Visual Studio extension to build, train, and deploy custom machine learning models.
```
Comment (luisquintanilla) [inline samples/modelbuilder/ObjectDetection_StopSigns/README.md:30]:
```suggestion
[Model Builder](https://docs.microsoft.com/dotnet/machine-learning/how-to-guides/install-model-builder) is an intuitive graphical Visual Studio extension to build, train, and deploy custom machine learning models.
```

File: samples/modelbuilder/ObjectDetection_StopSigns/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+## Goal
+
+Create a C# .NET Core Console application that uses an object detection machine learning model trained using Model Builder to determine if there is a stop sign within the image and give its bounding boxes.
+
+![Console output](./images/console-output.png)
+
+## Application
+
+- StopSignDetection:  A C# .NET Core Console application that uses an ML.NET model to find stop signs in images and, if any are found, returns their bounding boxes to where they are located in the image.
+- StopSignDetectionML.ConsoleApp: A .NET Core Console application that contains the model training and test prediction code.
```
Comment (luisquintanilla) [inline samples/modelbuilder/ObjectDetection_StopSigns/README.md:18]:
```suggestion
- **StopSignDetection**:  A C# .NET Core Console application that uses an ML.NET model to find stop signs in images and, if any are found, returns their bounding boxes to where they are located in the image.
- **StopSignDetectionML.ConsoleApp**: A .NET Core Console application that contains the model training and test prediction code.
```

**Review by luisquintanilla — COMMENTED** [review]
Minor nits

**Pull request merged**